blocks/library/button/index.js

```diff
@@ -228,9 +230,16 @@ registerBlockType( 'core/button', {
 	save( { attributes } ) {
```

Comment:
Changing the `save` function means the old button blocks will become invalid, and we should declare a "depreacted version" to automatically upgrade these old blocks. (See the quote block as an example). Unfortunately, this API is not yet documented.

Reply:
I can try to check it after holidays.

Reply:
In ad4629412e99b79f18dbabbeee0bbb475247bb6a I tried to declare a "deprecated version". I'm not sure how I could test this.

Reply:
You can test this by pasting in the code editor, the old markup. It should be converted to the new markup as soon as you focus out of the textarea.